indico/modules/events/registration/templates/emails/custom_email.html

```diff
@@ -4,5 +4,5 @@
 {% block footer %}{% endblock %}
 
 {% block body %}
-    {{ email_body | sanitize_html }}
+    {{ email_body | sanitize_html(allow_cid=True) }}
```

Comment:
While both work, the preferred capitalization for True/False/None in Jinja is lowercase.

```suggestion
    {{ email_body | sanitize_html(allow_cid=true) }}
```